src/test/integration/configuration.test.ts

```diff
@@ -0,0 +1,26 @@
+import { strictEqual } from 'assert';
+import { configuration } from '../../snyk/configuration';
+
+suite('Configuration', () => {
+  test('configuration constants differ between DEV and PROD', () => {
+    process.env.SNYK_VSCE_DEVELOPMENT = '';
+    strictEqual(configuration.baseURL, 'https://deeproxy.snyk.io');
+    strictEqual(configuration.authHost, 'https://snyk.io');
+
+    process.env.SNYK_VSCE_DEVELOPMENT = '1';
+    strictEqual(configuration.baseURL, 'https://deeproxy.dev.snyk.io');
+    strictEqual(configuration.authHost, 'https://dev.snyk.io');
+  });
+
+  test('configuration change is reflected', async () => {
+    const token = '17db2278-8d4c-4fb7-85c4-604721aa991b';
```

Comment:
This token is some dummy token?

Reply:
Yes 👍

Reply:
neat pick: token scares :), maybe use something like `12341234-abcd-abcd-abcd-123412341234`? Then it will be clear it's a dummy one.